src/scrape_up/flipkart/flipkart_laptop.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import bs4
+from bs4 import BeautifulSoup as b
+
+
+class flipkart_laptop:
+    """
+    Create an instance of `flipkart_laptop` class.
+    ```python
+    item = flipkart_laptops
+    ```
+    | Methods               | Details                                                            |
+    | --------------------- | ------------------------------------------------------------------ |
+    | `.laptops()`          | Returns the list of Laptops on flipkart with all the details       |
+    """
+
+    def __init__(self):
+        pass
+
+    def laptops(self):
```

Comment:
The method to fetch laptops already exists in the `flipkart_file.py`. There's not much difference between the code that already exists and the method you are adding now. 

See if you can add some extra, if not then close the PR it doesn't make sense to add the same code in the codebase. I'll recommend you try scraping details of a specific laptop by name, like screen size, ram, memory, etc. that you find in the specification section.